BraveRewardsUI/Tipping/TippingViewController.swift

```diff
@@ -25,17 +25,18 @@ enum PublisherMediaType: String {
 }
 
 class TippingViewController: UIViewController, UIViewControllerTransitioningDelegate {
-  
+  private var ledgerObserver: LedgerObserver
   let state: RewardsState
   let publisherInfo: PublisherInfo
   static let defaultTippingAmounts = [1.0, 5.0, 10.0]
   
   init(state: RewardsState, publisherInfo: PublisherInfo) {
     self.state = state
     self.publisherInfo = publisherInfo
-    
+    ledgerObserver = LedgerObserver(ledger: state.ledger)
+    state.ledger.add(ledgerObserver)
     super.init(nibName: nil, bundle: nil)
-    
+    setupLedgerObservers()
```

Comment:
We're going to handle showing confirmation based on callbacks added in #143. Unfortunately we won't want to wait for reconcile or observe recurring tips being added since we won't know when a reconcile will complete (could never happen) and 2. technically but likley never going to happen: other recurring tips could be added that are unrelated

Whether or not the tip goes out instantly, or its just added to pending contributions table, is dependent on a bunch of factors

So likely just going to take out all observer stuff in `TippingViewController`